Assets/_PackageRoot/Scripts/StoreSO.cs

```diff
@@ -100,6 +110,7 @@ List<StoreSellable>								SellablesList
 												public					bool											ValidateCurrency		(string currency)   => currency == null ? false : AllCurrenciesNames().Contains(currency);
 												public					bool											ValidateCategory		(string category)	=> category == null ? false : categories.ContainsKey(category);
 
+												public					bool											IsProductApplePromotional					=> _productApplePromotional != null;
```

Comment:
Looks like we don't really need the property `IsProductApplePromotional` with the variable `_productApplePromotional` at all. Because the promotional is available to receive from related events. If so, need to delete the property + variable.